packages/common/driver-definitions/src/storage.ts

```diff
@@ -148,7 +148,7 @@ export interface IDocumentStorageService extends Partial<IDisposable> {
 	/**
 	 * Policies implemented/instructed by driver.
 	 */
-	readonly policies?: IDocumentStorageServicePolicies;
+	readonly policies?: IDocumentStorageServicePolicies | undefined;
```

Comment:
This one might be better as mandatory (and update the other implementations to return undefined), as it's weird to have a public class member as optional.

```suggestion
	readonly policies: IDocumentStorageServicePolicies | undefined;
```

In the implementations:

```ts
public get policies() {
	return undefined;
}
```